pkgs/development/tools/misc/cli11/default.nix

```diff
@@ -9,27 +10,21 @@
 
 stdenv.mkDerivation rec {
   pname = "cli11";
-  version = "1.9.1";
+  version = "2.2.0";
 
   src = fetchFromGitHub {
     owner = "CLIUtils";
     repo = "CLI11";
     rev = "v${version}";
-    sha256 = "0hbch0vk8irgmiaxnfqlqys65v1770rxxdfn3d23m2vqyjh0j9l6";
+    sha256 = "sha256-emTIaoUyTINbAAn9tw1r3zLTQt58N8A1zoP+0y41yKo=";
   };
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake catch2 ];
```

Comment:
You are sure about this?

Reply:
Catch2 is needed to build the tests. I am a nix beginner so i can't say that I am sure where to put such dependencies.

Reply:
I moved `catch2` into `checkInputs` which seems to be the correct place?